.babelrc

```diff
@@ -2,14 +2,21 @@
   "presets": [
     [
       "@babel/env",
-      {"targets": {"node":"current"}}
+      {
+        "targets": {
+          "node": "8",
```

Comment:
I think we'll be using features only in current LTS and beyond (eg: bigint)

Reply:
oh yeah, >=10.4.0 supports bigint

Although, I would still keep node 8 support for libraries without bigint. What do you think?

Reply:
:+1: Yeah, as much support as possible